attalos/imgtxt_algorithms/regress2sum/regress2sum.py

```diff
@@ -87,7 +87,7 @@ def train_model(train_dataset,
     # Extract features and place in numpy matrix
     for i in test_dataset:
         image_feats, tags = test_dataset[i]
-        val_image_feats[i, :] = image_feats/np.linalg.norm(image_feats)
+        val_image_feats[i, :] = image_feats
```

Comment:
Yes, I had removed it from the training and failed to remove it from the test data. I found that it did not improve performance.